apps/condo/domains/user/components/auth/RegisterForm.tsx

```diff
@@ -154,6 +162,27 @@ export const RegisterForm: React.FC<RegisterFormProps> = ({ onReset, onFinish })
         }
     }, [checkUserExistenceQuery, executeCaptcha, isLoading, token])
 
+    const startConfirmEmailAction = useCallback(async (email) => {
+        try {
+            const sender = getClientSideSenderInfo()
+            const captcha = await executeCaptcha()
+
+            await startConfirmEmailActionMutation({
+                variables: {
+                    data: {
+                        dv: 1,
+                        sender,
+                        captcha,
+                        email: email,
+                        messageType: ConfirmEmailActionMessageType.VerifyUserEmail,
+                    },
+                },
+            })
+        } catch (error) {
+            console.log('Cannot start a user email verifying')
```

Comment:
shouldnt we show a toast?

Reply:
1) If the request is unsuccessful, there will already show toast. 
2) Any error shows an error screen.